docs/user-interface/controls/timepicker.md

```diff
@@ -67,3 +67,25 @@ However, this is not recommended. Depending on the setting of the `Format` prope
 <!--
 > [!TIP]
 > On Android, the `TimePicker` dialog can be customized by overriding the `CreateTimePickerDialog` method in a custom renderer. This allows, for example, additional buttons to be added to the dialog. -->
+
+## Platform differences
+
+This section describes the platform-specific differences with the `TimePicker` control.
+
+<!-- markdownlint-disable MD025 -->
+<!-- markdownlint-disable MD024 -->
+# [Android](#tab/android)
+
+On Android, the `Format` property is respected and displayed by the control. However, when the picker control is shown by pressing on the control, only the hour, minute, and time of day can be changed.
+
+# [iOS\macOS](#tab/ios)
+
+<!-- David, maybe you can fill this out? -->
+
+# [Windows](#tab/windows)
+
+On Windows, the `Format` property doesn't affect what is displayed by the control. When the picker is shown by pressing on the control, only the hour, minute, and time of day can be changed. For more information about the Windows control, see [Time picker - Windows apps](/windows/apps/design/controls/time-picker).
```

Comment:
Looking at the source code, `MapFormat` (https://github.com/dotnet/maui/blob/aba262370f299fe685250e3d1fa870f65f821a8a/src/Core/src/Handlers/TimePicker/TimePickerHandler.Windows.cs#L21) calls `UpdateTime` (https://github.com/dotnet/maui/blob/aba262370f299fe685250e3d1fa870f65f821a8a/src/Core/src/Platform/Windows/TimePickerExtensions.cs#L13) which says the format should switch between 24/12 hour clock.

Reply:
Good catch